src/ugeneui/src/Main.cpp

```diff
@@ -214,6 +218,20 @@ static void setDataSearchPaths() {
         dataSearchPaths.push_back(appDirPath + RELATIVE_DATA_DIR);
     } else if (QDir(appDirPath + RELATIVE_DEV_DATA_DIR).exists()) {    //data location for developers
         dataSearchPaths.push_back(appDirPath + RELATIVE_DEV_DATA_DIR);
+#ifdef Q_OS_MAC
+    } else {
+        CFURLRef appUrlRef = CFBundleCopyBundleURL(CFBundleGetMainBundle());
```

Comment:
nit: you can use the pattern here with translations code: get the path from U2Private. I think this part is the same with ugenecl & ugeneui code so you will have less code-duplication

Reply:
done